jupyter_book/__init__.py

```diff
@@ -23,6 +23,7 @@ def add_static_files(app, config):
 def setup(app):
 
     app.add_config_value("use_jupyterbook_latex", True, "env")
+    app.add_config_value("use_multitoc_numbering", True, "env")
```

Comment:
Wouldn't this be added by the sphinx extension rather than here?

Reply:
the `use_multitoc_numbering` value is set to determine if we add `sphinx_multitoc_numbering` to the sphinx extensions list.

Reply:
@choldgraf or are you saying this is already set by `default_config.py`?

Reply:
Ah, but if people are activating this extension with a config value in html, then we don't need to use sphinx to add the extension right?

Reply:
the extension will be added by `jupyter-book` by default. So the only `config` from a user will be to turn it off (ie. not loading the extension) should they not want continuous numbering across `parts`.